src/types/file.types.ts

```diff
@@ -12,54 +13,63 @@ export enum LicenseNames {
   CC0 = 'CC0',
 }
 
-export interface LicenseInfo {
+interface LicenseInfo {
   identifier: LicenseNames;
+  label: string;
   description: string;
   image: any;
+  buttonImage: any;
 }
 
 export const licenses: LicenseInfo[] = [
   {
     identifier: LicenseNames.CC_BY,
+    label: i18n.t('licenses:labels.cc_by'),
     image: LicenseImages.ccByImage,
-    description:
-      'This license lets others distribute, remix, adapt, and build upon your work, even commercially, as long as they credit you for the original creation. This is the most accommodating of licenses offered. Recommended for maximum dissemination and use of licensed materials.',
+    buttonImage: LicenseImages.ccByButton,
```

Comment:
hvorfor trenger vi egentlig å skille på image og buttonImage? skal det ikke være samme bilde med ulik størrelse?

Reply:
Det er to forskjellige ikon. 
Den ene samlingen er for buttons (de finnes allerede ferdiglaget):
![Skjermbilde 2020-05-14 kl  08 12 47](https://user-images.githubusercontent.com/6595982/81899281-c0d71c00-95ba-11ea-87fe-0b8c85595e20.png)

Den andre samlingen har jeg laget selv ved å sette sammen de forskjellige sirklene:
![Skjermbilde 2020-05-14 kl  08 12 59](https://user-images.githubusercontent.com/6595982/81899331-d8aea000-95ba-11ea-9e5d-4633fe2b15a2.png)